### DISCUSSION TITLE
Uno-check run into multiple error

### DISCUSSION THREAD

**Discussion by adssib** [unoplatform/uno#13697] [Q&A]
Hello, I have been trying to start with the Uno-Platform.
I followed the instructions on the website to set up my environment, However while running uno-check in the cmd there is some errors that are not solved no matter how many times i have tried to solve them.

Here are the screenshots: 
![image](https://github.com/unoplatform/uno/assets/75389300/00e867b8-ed5c-4e2b-acff-4eccfc23d496)
![image](https://github.com/unoplatform/uno/assets/75389300/7a9c445c-0958-434f-99d4-022c40d033b4)
![image](https://github.com/unoplatform/uno/assets/75389300/1885634f-f843-4de6-b5c0-14d8d49c0618)

**Comment by jeromelaban** [comment]
Thanks for the report. Did you check the error file mentioned in the logs?

**Reply by jeromelaban** [reply]
Also, it only seems to happen because you already have a later version of the .NET SDK. We'll take a closer look at why this happens. In the meantime, you can skip the .NET SDK installation, and let the workloads installation continue.

**Reply by adssib** [reply]
I am sorry Where do i find this?

Keep in mind that I am just opening cmd till now without opening VS

**Reply by jeromelaban** [reply]
The message says "see log file for more details", you can try opening that file.

**Reply by adssib** [reply]
Ok i found the log file.
[logFile.txt](https://github.com/unoplatform/uno/files/12662208/logFile.txt)
